### ISSUE TITLE
-S only works after package name for yarn add

### ISSUE THREAD

**Issue by OmgImAlexis** [yarnpkg/yarn#3128]
**Do you want to request a *feature* or report a *bug*?**
Bug
**What is the current behavior?**
```
➜  wvvw.me git:(master) ✗ yarn add -S cz 
yarn add v0.21.3
error Missing list of packages to add to your project.
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
```

**If the current behavior is a bug, please provide the steps to reproduce.**
```
yarn add -S cz
```

**What is the expected behavior?**
The `-S` flag should work before or after the package name just like `-D` does.

**Please mention your node.js, yarn and operating system version.**
npm version 4.1.2
yarn version v0.21.3
Mac OS 10.12.3 (16D32)

This happens on Bash and ZSH for me.

**Comment by smitt04** [comment]
There is no `-S` for `add`. What do you expect `-S` to do? if it is the same as `npm` and to save package to `package.json`, then add does that by default

**Comment by OmgImAlexis** [comment]
Yes, since there's a `-D` flag I would expect `-S` to work the same as it does. 
Even if it doesn't do anything it shouldn't cause an error. Coming from npm I assumed `-D` was the same as `--save-dev` and `-S` was the same as `--save`.
